python3-kolibri-app-desktop-xdg-plugin.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "python3-kolibri-app-desktop-xdg-plugin",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} kolibri-app-desktop-xdg-plugin"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/60/f0/dd2eb7911f948bf529f58f0c7931f6f6466f711bd6f1d81a69dc4edd4e2a/Pillow-8.1.2.tar.gz",
+            "sha256": "b07c660e014852d98a00a91adfbe25033898a9d90a8f39beb2437d22a203fc44"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/a4/ef/68a14ab852619b12016eefda09198facdb90a4ceeec04ad737a74b79f806/kolibri_app_desktop_xdg_plugin-1.0.1-py3-none-any.whl",
```

Comment:
The current release of kolibri-app-desktop-xdg-plugin in pypi.org is broken. At least in my tests I've detected that importing a channel raises an exception there.

With the current version on master it's working correctly and the icons are created on channel import.